### PR TITLE
CI: add apisix-docker test

### PR DESCRIPTION
**Pull request by gxthrj**
- [x] add docker test CI
- [x] upload the image tar

### CODE REVIEW

File: .github/workflows/apisix-alpine-docker-test.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          ALLOW_NONE_AUTHENTICATION: yes
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build and Test
+        run: |
+          docker build -t apisix:alpine -f ./alpine/Dockerfile .
+          docker run -v `pwd`/all-in-one/apisix/config.yaml:/usr/local/apisix/conf/config.yaml -p 9080:9080 -d apisix:alpine
+          sleep 30
+          curl http://127.0.0.1:9080/apisix/admin/schema/service -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1'
```
[the remark this answers is not in the document]
Comment (gxthrj) [reply .github/workflows/apisix-alpine-docker-test.yaml:26]: View the returned data for the first time.

File: .github/workflows/apisix-alpine-docker-test.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: apisix Alpine docker Test
```
[the remark this answers is not in the document]
Comment (gxthrj) [reply .github/workflows/apisix-alpine-docker-test.yaml]: Done

File: example/docker-compose.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   apisix:
-    image: apache/apisix:2.3-alpine
+    image: apache/apisix:2.3-centos
```
[the remark this answers is not in the document]
Comment (gxthrj) [reply example/docker-compose.yml]: Revert

File: .github/workflows/apisix-alpine-docker-test.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tokers) [reply .github/workflows/apisix-alpine-docker-test.yaml:26]: I think it's unnecessary in CI.

[the remark this answers is not in the document]
Comment (starsz) [reply .github/workflows/apisix-alpine-docker-test.yaml:26]: Is it for debug? If so, we can delete it.

File: .github/workflows/apisix-alpine-docker-test.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+          docker build -t apisix:alpine -f ./alpine/Dockerfile .
+          docker-compose -f ./alpine/docker-compose.yaml up -d
+          sleep 30
+          docker logs docker_apisix_1
+
+      - name: Test
+        run: |
+          curl http://127.0.0.1:9080/apisix/admin/schema/service -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1'
+          code=$(curl -k -i -m 20 -o /dev/null -s -w %{http_code} http://127.0.0.1:9080/apisix/admin/schema/service -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1')
+          if [ ! $code -eq 200 ]; then
```
Comment (starsz) [inline .github/workflows/apisix-alpine-docker-test.yaml:28]: IMO, the test is a little bit easy.

File: .github/workflows/apisix-alpine-docker-test.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (gxthrj) [reply .github/workflows/apisix-alpine-docker-test.yaml:26]: If there is something wrong, the return data can help us to locate the problem quickly.

File: .github/workflows/apisix-alpine-docker-test.yaml (hunk 4 above)
Comment (gxthrj) [reply .github/workflows/apisix-alpine-docker-test.yaml:28]: Just a smoking here

Comment (starsz) [reply .github/workflows/apisix-alpine-docker-test.yaml:28]: Yes. But I think we can test the route.
Refer: https://github.com/api7/apisix-build-tools/pull/14/files#diff-1e94407ae9a7cfa3661dbeeff2953068de2c2b999abb124e26ab7fd62830fb13

Comment (gxthrj) [reply .github/workflows/apisix-alpine-docker-test.yaml:27]: OK, will change to route test

**Review by tokers — APPROVED** [review]

**Review by membphis — APPROVED** [review]

File: .github/workflows/apisix-alpine-docker-test.yaml (hunk 4 above)
Comment (starsz) [reply .github/workflows/apisix-alpine-docker-test.yaml:27]: How is it going?

File: .github/workflows/apisix-alpine-docker-test.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (gxthrj) [reply .github/workflows/apisix-alpine-docker-test.yaml]: Have changed to route test

File: .github/workflows/apisix-alpine-docker-test.yaml (hunk 4 above)
Comment (gxthrj) [reply .github/workflows/apisix-alpine-docker-test.yaml]: Done

**Review by starsz — APPROVED** [review]

**Pull request merged**